arrange_schedule.py

```diff
@@ -434,7 +434,7 @@ def edit_schedule(json_obj):
             if target_sn:
                 #check use update or insert
                 with ScheduleDao() as scheduleDao:
-                    sche_sn = scheduleDao.getEditScheSn(scheSn=target_sn)
+                    sche_sn = scheduleDao.getEditScheSn(scheSn=target_sn+next_sn)
```

Comment:
I think we can change `next_sn` to `next_sn_offset`, because the original name will mislead us to comprehend the meaning of this variable.